spec/system/teacher_interface/application_spec.rb

```diff
@@ -685,266 +710,482 @@ def and_i_receive_an_application_email
+      "Your reference number"
+    )
+    expect(submitted_application_page.reference).to have_content(
+      application_form.reference
+    )
   end
 
   def then_i_see_delete_confirmation_form
-    expect(page).to have_title("Delete")
-    expect(page).to have_content("Are you sure you want to delete")
+    expect(delete_confirmation_page).to have_title("Delete")
+    expect(delete_confirmation_page.heading).to have_content(
+      "Are you sure you want to delete"
+    )
+  end
+
+  def eligible_page
+    @eligible_page = PageObjects::EligibilityInterface::Eligible.new
+  end
+
+  def active_application_page
+    @active_application_page =
+      PageObjects::TeacherInterface::ActiveApplication.new
+  end
+
+  def new_teacher_form_page
+    @new_teacher_form_page =
+      PageObjects::TeacherInterface::NewApplicationForm.new
+  end
+
+  def subjects_form_page
+    @subjects_form_page = PageObjects::TeacherInterface::SubjectsForm.new
+  end
+
+  def work_history_form_page
+    @work_history_form_page = PageObjects::TeacherInterface::WorkHistoryForm.new
+  end
+
+  def delete_confirmation_page
+    @delete_confirmation_page =
+      PageObjects::TeacherInterface::DeleteConfirmationForm.new
+  end
+
+  def written_statement_page
+    @written_statement_page =
+      PageObjects::TeacherInterface::WrittenStatement.new
+  end
+
+  def personal_information_summary_page
+    @personal_information_summary_page =
+      PageObjects::TeacherInterface::PersonalInformationSummary.new
+  end
+
+  def qualification_summary_page
+    @qualification_summary_page =
+      PageObjects::TeacherInterface::QualificationSummary.new
+  end
+
+  def qualifications_form_page
+    @qualifications_form_page =
+      PageObjects::TeacherInterface::QualificationsForm.new
+  end
+
+  def name_and_date_of_birth_page
+    @name_and_date_of_birth_page =
+      PageObjects::TeacherInterface::NameAndDateOfBirth.new
+  end
+
+  def check_your_answers_page
+    @check_your_answers_page =
+      PageObjects::TeacherInterface::CheckYourAnswers.new
+  end
+
+  def work_history_summary_page
+    @work_history_summary_page =
+      PageObjects::TeacherInterface::WorkHistorySummary.new
+  end
+
+  def upload_document_page
+    @upload_document_page =
+      PageObjects::TeacherInterface::UploadDocumentForm.new
+  end
+
+  def submitted_application_page
+    @submitted_application_page =
+      PageObjects::TeacherInterface::SubmittedApplication.new
+  end
+
+  def check_your_uploads_page
+    @check_your_uploads_page =
+      PageObjects::TeacherInterface::CheckYourUploads.new
+  end
+
+  def alternative_name_page
+    @alternative_name_page =
+      PageObjects::TeacherInterface::AlternativeNameForm.new
+  end
+
+  def age_range_form
+    @age_range_page = PageObjects::TeacherInterface::AgeRangeForm.new
+  end
+
+  def registration_number_form
+    @registration_number_form =
+      PageObjects::TeacherInterface::RegistrationNumberForm.new
   end
```

Comment:
These can go in the `page_helpers.rb` file as well, the idea is all pages objects will be defined in this one place so the individual spec files can be focussed on the tests.